grafana/README.md

```diff
@@ -0,0 +1,22 @@
+# Grafana
+
+This directory contains useful out-of-box dashboards.
+
+# Constraints
+
+Dashboards contain [summary](../grafana/dashboards/summary) and [single](../grafana/dashboards/single) dashboards.
+* summary: Aggregated dashboards for a cluster.
+* single: dashboards for a single resource(e.g. `node` or `pod`).
+
+**Notice**
+* summary dashboards use `alias` as the `cluster` template variable.
```

Comment:
From what I've seen people put their cluster name in the `external_labels` in Prometheus. So generally yes I think it's sensible to have a cluster selection, but also have the dashboard not care if there is only one cluster even if the metrics don't have that label.